### DISCUSSION TITLE
After your last release, the plugin no long protect my website

### DISCUSSION THREAD

**Discussion by monsieurdd** [verbb/knock-knock#65] [General]
FYI,
After your last release 2.0.9, the plugin no long seems to protects my website or do anything in Craft 4 last version.
- I've clear cache,
- reinstall
- try changing password
- Connected with a VPN external to see and try accessing the website to see

and still enable not protected when everything is enabled.

**Comment by engram-design** [comment]
FYI, this shouldn't be a discussion, so moving to https://github.com/verbb/knock-knock/issues/66
